static/js/src/imageBuilder.js

```diff
@@ -310,10 +343,22 @@
   function checkDisabled() {
     step2.classList.add("u-disable");
     step3.classList.add("u-disable");
-    if (state.get("board") && state.get("board")[0] != "") {
+    if (state.get("board") && state.get("board")[0]) {
```

Comment:
Can simplify with optional chaining - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Array_item_access_with_optional_chaining

```suggestion
    if (state.get("board")?.[0]) {
```

Reply:
I think that is much harder to read. Isn't it better to keep the code readable over shorter?

Reply:
Optional chaining has been introduced to the spec specifically to replace that hack of truthy short circuit evaluation. As such, it will become _the_ correct way to check if something exists before using it.

It looks weird the first couple of times you see it but then you see duplication everywhere else that can be eradicated by using it.

Reply:
I tried to add it but eslint is failing with:
`360:28  error  Parsing error: Unexpected token .`